ui/pageTemplate.glade

```diff
@@ -1,308 +1,301 @@
+              <packing>
+                <property name="expand">True</property>
+                <property name="fill">True</property>
+                <property name="position">0</property>
+              </packing>
+            </child>
+            <child>
+              <object class="GtkButton" id="btLoad">
+                <property name="label" translatable="yes">Load from file</property>
+                <property name="name">btLoad</property>
+                <property name="visible">True</property>
+                <property name="can-focus">True</property>
+                <property name="receives-default">False</property>
+              </object>
               <packing>
                 <property name="expand">True</property>
                 <property name="fill">True</property>
                 <property name="position">1</property>
               </packing>
             </child>
+            <child>
+              <object class="GtkButton" id="btCancel">
+                <property name="label" translatable="yes">Cancel</property>
+                <property name="name">btCancel</property>
+                <property name="visible">True</property>
+                <property name="can-focus">True</property>
+                <property name="can-default">True</property>
+                <property name="receives-default">True</property>
+                <property name="image">iconCancel</property>
+                <accelerator key="Escape" signal="clicked"/>
+              </object>
+              <packing>
+                <property name="expand">True</property>
+                <property name="fill">True</property>
+                <property name="position">2</property>
+              </packing>
+            </child>
+            <child>
+              <object class="GtkButton" id="btOk">
+                <property name="label" translatable="yes">Ok</property>
+                <property name="name">btOk</property>
+                <property name="visible">True</property>
+                <property name="can-focus">True</property>
+                <property name="receives-default">True</property>
+                <property name="image">iconOk</property>
+              </object>
+              <packing>
+                <property name="expand">True</property>
+                <property name="fill">True</property>
+                <property name="position">3</property>
+              </packing>
+            </child>
           </object>
           <packing>
-            <property name="expand">True</property>
+            <property name="expand">False</property>
             <property name="fill">True</property>
             <property name="position">2</property>
           </packing>
         </child>
       </object>
     </child>
-    <action-widgets>
-      <action-widget response="2">btCancel</action-widget>
-      <action-widget response="1">btOk</action-widget>
-    </action-widgets>
   </object>
 </interface>
```

Comment:
The GtkBox containing the buttons and the GtkBox containing everything don't have an ID. Not sure if that could ever be an issue, but as I recall everything in our Glade files had an ID so far. So maybe add one here, too.

Reply:
I'm not sure about that. I think that naming everything may be a mistake actually: I haven't looked under the hood, but I'd assume that GtkBuilder has a map id->GObject* of named objects. Uselessly giving widgets an id would just increase the map's size thus slowing down (even so slightly) the name-matching searches.
It's probably not worth removing the unused id, but why not avoid adding more?

Reply:
Your arguments make sense to me. So let's avoid adding unused id's in future.